src/main/java/subway/ui/SubwayControllerAdvice.java

```diff
@@ -34,4 +35,10 @@ public ResponseEntity<ErrorResponseDto> handleAddSectionException(final AddSecti
         final String message = exception.getMessage();
         return ResponseEntity.badRequest().body(new ErrorResponseDto(message));
     }
+
+    @ExceptionHandler(NoSuchStationException.class)
+    public ResponseEntity<ErrorResponseDto> handleNoSuchStationException(final NoSuchStationException exception) {
+        final String message = exception.getMessage();
+        return ResponseEntity.badRequest().body(new ErrorResponseDto(message));
```

Comment:
```suggestion
    @ExceptionHandler({SQLException.class, IllegalArgumentException.class, AddSectionException.class, NoSuchStationException.class})
    public ResponseEntity<ErrorResponseDto> badRequest(final Exception exception) {
        final String message = exception.getMessage();
        return ResponseEntity.badRequest().body(new ErrorResponseDto(message));
```

중복되는 예외 처리는 이렇게 묶을 수 있어요.